### PR TITLE
fixture of Contributor Loading

### PR DESCRIPTION
**Pull request by ghost**
Now our contributor loading page will show a template something like "Thanks for username's contribution". It seems this won't be shown until all the variables from the remote GitHub loaded successfully. So first hide the contributor card and then show it when ready.

### CODE REVIEW

File: layouts/partials/contributor-card.hbs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<div class="contributor-card">
+<div class="contributor-card" style="display: none;">
```
Comment (Trott) [inline layouts/partials/contributor-card.hbs:1]: Should this be `visibility: hidden` instead of `display: none`?

Comment (ghost) [reply layouts/partials/contributor-card.hbs:1]: I understand what you mean: No matter whether the data is loaded, the space should be left.
But it looks so ugly, so are you sure you want this? Something like:


IMO: If it fails to load, the space shouldn't be shown and take place until all the data is loaded successfully....

**Comment by XhmikosR** [comment]
This was intentional though, so that the page jump isn't that big.

File: layouts/partials/contributor-card.hbs (hunk 1 above)
Comment (Trott) [reply layouts/partials/contributor-card.hbs:1]: Ah, yeah, I can see the merit of `display:none` in that case.

**Pull request closed without merging**